content/docs/conditional-rendering.md

```diff
@@ -116,13 +116,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Coba di Codepen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Saat mendeklarasikan sebuah variabel dan  menggunakan statement `if` merupakan cara yang baik merender komponen secara kondisional, terkadang Anda mungkin ingin memakai sintaksis pendek. Ada beberapa cara _iniline_ bersyarat pada JSX, dijelaskan dibawah.
 
-### Inline If with Logical && Operator {#inline-if-with-logical--operator}
+### Inline If dengan Operator Logis && {#inline-if-with-logical--operator}
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Anda mungkin [menyisipkan ekspresi pada JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) dengan cara membungkusnya ke dalam kurung kurawal. Juga memasukan operator logis `&&`. Itu dapat berguna untuk memasukan elemen secara kondisional:
```

Comment:
Maybe this, which rephrase the usage of curly braces, and also added `apapun`:

```suggestion
Anda dapat [menyisipkan ekspresi apapun pada JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) dengan cara membungkusnya ke dalam kurung kurawal. Juga memasukan operator logis `&&`. Kurung kurawal dapat berguna untuk memasukan elemen secara kondisional:
```

Also, this part of sentence should be discussed further:

> dengan cara membungkusnya ke dalam kurung kurawal